### PR TITLE
pass arches, when requesting module compose

### PR DESCRIPTION
**Pull request by rcerven**
Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

File: tests/plugins/test_resolve_composes.py [hunk 1]
```diff
@@ -285,9 +289,10 @@ def test_request_compose_for_modules(self, workflow, reactor_config_map):
             .with_args(
                 source_type='module',
                 source='spam bacon eggs',
-                sigkeys=['R123'])
+                sigkeys=['R123'],
+                arches=arches)
             .and_return(ODCS_COMPOSE))
```
Comment (athos-ribeiro) [inline tests/plugins/test_resolve_composes.py]: Should the return value still contain only x86_64 here?

File: tests/plugins/test_resolve_composes.py [hunk 2]
```diff
@@ -254,7 +254,7 @@ def test_request_compose(self, workflow, reactor_config_map):
         self.run_plugin_with_args(workflow, reactor_config_map=reactor_config_map)
 
     @pytest.mark.parametrize('arches', (
-        ['x86_64', 'ppc64le'],
+        ['ppc64le', 'x86_64'],
```
Comment (athos-ribeiro) [inline tests/plugins/test_resolve_composes.py]: Was this change needed here for some reason or is it just a consequence of the 'parametrize' in line 273 for `test_request_compose_for_modules`?

Comment (rcerven) [reply tests/plugins/test_resolve_composes.py]: test wasn't working as intended, to define arches for resolve_compose, arches has to be in check_and_set_platforms plugin, ALSO odcsclient is mocked also in workflow, so it was basically ignoring our mock with arches, I am adding here once() so it is more clearer

Comment (athos-ribeiro) [reply tests/plugins/test_resolve_composes.py]: Thanks for the reply

**Comment by athos-ribeiro** [comment]
@rcerven Python 3 test suite was deactivated. Feel free to rebase and merge

**Comment by athos-ribeiro** [comment]
Release notes added:
# Bug Fixes
- Request architectures on module compose requests
